doc/api/events.markdown

```diff
@@ -162,3 +162,22 @@ added.
 This event is emitted *after* a listener is removed.  When this event is
 triggered, the listener has been removed from the array of listeners for the
 `event`.
+
+### Inheriting from 'EventEmitter'
+
+Inheriting from `EventEmitter` is no different from inheriting from any other
+constructor function. For example:
+
+```js
```

Comment:
I'm not sure if this works with our parser. (I think that's why we still use 4-space indentation? I'm not too sure.)